frontend/src/containers/DocumentList.js

```diff
@@ -209,10 +209,8 @@ class DocumentListContainer extends Component {
       fetchQuickActions,
       isModal,
       viewProfileId,
-      viewData: layout,
     } = this.props;
     const viewId = customViewId ? customViewId : this.props.viewId;
-    const { uncollapseRowsOnChange } = layout;
```

Comment:
why not considering the `uncollapseRowsOnChange` from layout?

Reply:
oh. You still want that to be added ? Np.  I can added back.